metrics-operator/controllers/common/providers/prometheus/prometheus.go

```diff
@@ -3,35 +3,38 @@ package prometheus
 import (
 	"context"
 	"encoding/json"
-	"fmt"      //nolint:gci
```

Comment:
Is it possible to remove the nolint comments?

Reply:
for sure again a copypasta